examples/knative/powershell/kn-ps-vrni-databus/test/docker-test-env-variable

```diff
@@ -0,0 +1 @@
+WEBHOOK_CONFIG={"WEBHOOK_SINK_URL": ""}
```

Comment:
nit: as the value you might want to use FILL_ME_IN so it's clear during debugging and the user sees the issue instead of having a blank/zero value